### ISSUE TITLE
cmd/ctrl-click on import identifier doesn't work on npm packages

### ISSUE THREAD

**Issue by shirotech** [intellij-solidity/intellij-solidity#402]
For example:

```solidity
import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
import {Ownable} from "./Ownable.sol";
```

cmd-click doesn't work when clicking on `{Initializable}` or `{IERC20}`, but it works for local files in the example `{Ownable}`. cmd-click on the filePath is fine though, to make sure that the file actually exists.

**Comment by jonahcui** [comment]
Are you sure that the "import x from x" statement is correct? Does the solidity support the grammar? 

[The Solidity Doc](https://remix-ide.readthedocs.io/en/latest/import.html)

**Comment by shirotech** [comment]
> Are you sure that the "import x from x" statement is correct? Does the solidity support the grammar?
> 
> [The Solidity Doc](https://remix-ide.readthedocs.io/en/latest/import.html)

Yes, I'm 100% sure it is supported, otherwise it would complain about compilation errors. Not sure why you referenced a 3rd party docs that may not have the complete set of rules. Please see https://docs.soliditylang.org/en/latest/layout-of-source-files.html for the official reference, thank you 🙏

**Comment by jonahcui** [comment]
<img width="1034" alt="CleanShot 2023-09-15 at 16 37 03@2x" src="https://github.com/intellij-solidity/intellij-solidity/assets/101780038/fc6c18a8-1927-46de-90a7-7bb7580e8976">

Can you share your project structure? I found the reference is work correctly.

**Comment by shirotech** [comment]
@jonahcui That looks like just the docs for that file, what happens when you cmd+click on the text between the {} ? The expected behaviour is that it should jump to the file with scrollbar targeted at the class name.

**Comment by jonahcui** [comment]
It will jump to the code source.
2023年9月15日 +0800 17:10 Van Nguyen ***@***.***>，写道：

**Comment by XAudit174176** [comment]
> For example:
> 
> ```solidity
> import {Initializable} from "@openzeppelin/contracts/proxy/utils/Initializable.sol";
> import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
> import {Ownable} from "./Ownable.sol";
> ```
> 
> cmd-click doesn't work when clicking on `{Initializable}` or `{IERC20}`, but it works for local files in the example `{Ownable}`. cmd-click on the filePath is fine though, to make sure that the file actually exists.

Hi there,

Looks like you have a problem with the external imports, please consider these 2 things:

1. The external contracts files may not be downloaded or maybe corrupt, check if the file exists in the exact path, or even try reinstalling the dependencies
2. If the file exists, you need to check if you have defined appropriate remappings in order, so that the compiler and extension could understand for example `@openzeppelin/contracts/` should be replaced by `lib/openzeppelin-contracts/contracts/`


Btw if the compiler is doing well, the extension also should work

Also the problem doesn't seem to be general, as it is Ok now on my system (current new version by the time of submitting this response) !

Good Luck

**Comment by SerCeMan** [comment]
Hi, @shirotech! I believe this has likely been resolved by adding support for remappings, so I'll close this. If you're still experiencing the issue, please re-open it.
